app/src/store/stores/routerStore.ts

```diff
@@ -14,7 +14,7 @@ export default class RouterStore {
     this.history = history;
     this.location = history.location;
 
-    history.listen(location => {
+    history.listen(({ location }) => {
```

Comment:
The argument to the listener is [now](https://github.com/remix-run/history/blob/main/docs/api-reference.md#historylistenlistener-listener) the `Update` interface.